KalturaWowzaServer/release/build.xml

```diff
@@ -112,13 +112,35 @@
 		</propertyfile>
 	</target>
 
+	<target name="delete-old-version-files">
+		<!-- create fileset from all files in media-server lib directory-->
+		<fileset dir="${basedir}/lib" includes="*" id="lib-patterns"/>
+
+		<!-- remove version string to match older files-->
+                <property name="pattern-string" refid="lib-patterns"/>
+                <loadresource property="subst-pattern-string">
+                	<propertyresource name="pattern-string"/>
+                        <filterchain>
+                        	<tokenfilter>
+                                	<filetokenizer/>
+                                       	<containsregex pattern="-[0-9]+(\.[0-9]+)*" replace="*" flags="g"/>
```

Comment:
this regex is too general.
will it also include files such as wms-xstream-1.4.7.jar that you don't want to delete?